backend/debug/dwarf/dwarf_flags/dune

```diff
@@ -4,6 +4,6 @@
   (name dwarf_flags)
   (wrapped true)
   (flags (:standard -principal))
-  (ocamlopt_flags (:standard -O3))
+  (ocamlopt_flags (:standard))
```

Comment:
We should just delete the whole `ocamlopt_flags` clause (here and anywhere it's just `(:standard)` now).

Reply:
We should, but I think maybe in a follow-up PR (just makes it very clear what the changes are here)

Reply:
Sure, makes sense